tests/python/relay/test_external_runtime.py

```diff
@@ -492,8 +492,9 @@ def test_engine_extern():
                options=["-O2", "-std=c++11", "-I" + tmp_path.relpath("")])
 
 def test_json_extern():
-    if which("gcc") is None:
-        print("Skip test because gcc is not available.")
+    if not tvm.get_global_func("module.loadfile_examplejson", True):
+        print("Skip because JSON example runtime is not enabled.")
+        return
```

Comment:
Should have a `return` after L447?

Reply:
yeah, I didn't notice that one. Will add. Thanks.